actionview/lib/action_view/railtie.rb

```diff
@@ -96,6 +56,18 @@ class Railtie < Rails::Engine # :nodoc:
       PartialRenderer.collection_cache = app.config.action_controller.cache_store
     end
 
+    config.after_initialize do |app|
```

Comment:
Doing that inside an `after_initialize` so that application and libraries have time to change the `action_view. cache_template_loading`  config value if they need to